gemstone/common/mux_wrapper_aoi.py

```diff
@@ -38,13 +34,18 @@ def definition(io):
                     f = open("mux_aoi_const.sv", "w")
                     # 1-bit extra for the constant
                     num_sel = math.ceil(math.log(height + 1, 2))
+                    num_ops = math.ceil((height + 1) / 2)
                 else:
                     f = open("mux_aoi.sv", "w")
                     num_sel = math.ceil(math.log(height, 2))
+                    num_ops = math.ceil(height / 2)
                 num_inputs = math.pow(2, num_sel)
 
-                f.write(f"module mux_aoi_{height}_{width} ( \n")
-
+                # ======= MUX MODULE =========
+                if mux_type == mux_type.Const:
+                    f.write(f"module mux_aoi_const_{height}_{width} ( \n")
```

Comment:
This should be refactored to use a in-memory string rather than write to file. There is no need to write to file. We can use the function `DefineFromVerilog` rather than `DefineFromVerilogFile`

Reply:
Do you have an example?

Reply:
Moved these files to test/common/rtl folder based off discussion with @Kuree. Note these files are needed only for Fault tests and *not* for the design.

Reply:
That makes sense for those standard cell files. I'm talking about these `f.write` statements. These should just append to a string in memory rather than writing to file then reading from that file. Just extra file IO overhead we don't need.